CONTRIBUTING.md

```diff
@@ -15,52 +55,221 @@ simple statement that you, as a contributor, have the legal right to make the
+For bonus points, please highlight how feature requests/PRs fit into the vision when you can!
 
-- **important-longterm**: Important over the long term, but may not be currently staffed and/or may require multiple releases to complete.
+---
 
-- **backlog**: There appears to be general agreement that this would be good to have, but we may not have anyone available to work on it right now or in the immediate future. Community contributions would be most welcome in the meantime (although it might take a while to get them reviewed if reviewers are fully occupied with higher priority issues, for example immediately before a release).
+# Opening Issues
 
-- **awaiting-more-evidence**: Possibly useful, but not yet enough support to actually get it done. These are mostly place-holders for potentially good ideas, so that they don't get completely forgotten, and can be referenced /deduped every time they come up.
+These guides aim to help you write issues in a way which will ensure that they are processed
+as quickly as possible.
 
-These priority categories have been inspired by [the Kubernetes contributing guide](https://github.com/kubernetes/community/blob/master/contributors/guide/issue-triage.md).
+_See below for [how issues are prioritized](#prioritizing-issues)_.
+
+**General rules**:
+
+1. Before opening anything, take a good look through existing issues.
+1. More is more: give as much information as it is humanly possible to give.
+  Highly detailed issues are more likely to be picked up because they can be prioritized and
+  scheduled for work faster. They are also more accessible
+  to the community, meaning that you may not have to wait for the core team to get to it.
+1. Please do not open an issue with a description that is simply a link to another issue,
+  a link to a slack conversation, a quote from either one of those, or anything else
+  equally opaque. This raises the bar for entry and makes it hard for the community
+  to get involved. Take the time to write a proper description and summarise key points.
+1. Take care with formatting. Ensure the [markdown is tidy](https://docs.github.com/en/free-pro-team@latest/github/writing-on-github/getting-started-with-writing-and-formatting-on-github),
+  use [code blocks](https://docs.github.com/en/free-pro-team@latest/github/writing-on-github/creating-and-highlighting-code-blocks) etc etc.
+  The faster something can be read, the faster it can be dealt with.
+1. Keep it civil. Yes, it is annoying when things don't work, but it is way more fun helping out
+  someone who is not... the worst. Remember that conversing via text exacerbates
+  everyone's negativity bias, so throw in some emoji when in doubt :+1: :smiley: :rocket: :tada:.
+
+**Dedicated guides**:
+- [Bug report guide](#bug-report-guide)
+- [Feature request guide](#feature-request-guide)
+- [Help request guide](#help-request-guide)
+
+## Bug report guide
```

Comment:
shall we mention to add the `kind/bug` label to issue they open? Same for feature and help guides

Reply:
Happens automatically I thought? Only members have power to add labels

Reply:
oh does it? I hadn't observed that. I feel like I have also seen users add the label themselves